README.md

```diff
@@ -38,24 +38,38 @@ During this recording, we:
 
 [![demo](https://asciinema.org/a/230671.svg)](https://asciinema.org/a/230671?autoplay=1)
 
-## Pre-requirements
 
-- Python 3.8 or greater.
-- The Python3 binding for libvirt, the package is probably called `python3-libvirt`.
-- You can also just build the binding during the install, in this case, you will need to install some packages first:
-    - `dnf install libvirt-devel python3-devel` (Fedora/RHEL)
-    - `apt install libvirt-dev python3-dev` (Debian/Ubuntu)
-- You make also want to install `python3-urwid` if you want to get the fancy list of VM. This dependency is optional.
+## Requirements
+- 1Gb storage space
+- 100Mb ram
+- Python 3.8 or greater
+- The Python3 binding for libvirt, the package is probably called `python3-libvirt` or 'libvirt-python' according to pip.
 - Libvirt must be running, most of the time you just need to run: `sudo systemctl start --now libvirtd`
 - Finally, be sure your user can access the system libvirt daemon, e.g with: `virsh -c qemu:///system`
+- *** (Debian/Ubuntu) pipx, and additional packages to build virt-lightning ***
+
+## Optional
+- You make also want to install `python3-urwid` if you want to get the fancy list of VM. This dependency is optional.
 
-## Installation
 
+## Installation (Fedora/RHEL)
 ```shell
+dnf install libvirt-devel python3-devel
```

Comment:
```shell
dnf install pipx libvirt-devel python3-devel
pipx install virt-lightning
```

Reply:
I didn't investigate/test pipx on REHL/Fedora.  If the desire is for a unified pipx install for all platforms, I'll test that, I suspect it'll also need some dependency's.

Reply:
I tested it, it works as expected. :-)